app/models/micropost.rb

```diff
@@ -0,0 +1,18 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+
+  scope :order_time, ->{order created_at: :desc}
+
+  mount_uploader :picture, PictureUploader
+
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
``` 
2 validate này khác gì nhau?

Reply:
> Suggested change 
>   validates :user_id, presence: true
>   validates :user, presence: true
> Commit suggestion 
> Add suggestion to batch
> 2 validate này khác gì nhau?

em ko chắc với câu trả lời này lắm.
theo em với validates user_id tương ứng với việc rằng buộc user_id trong bảng microposts với điều kiện presence true nghĩa là ko được trống.
còn validates user thì là vì do đã kết nối với bảng user với kết nối 1-n từ user - microposts qua foreign key là user_id nên validates user theo em hiểu nó đang cần điều kiện gì đó từ bên bảng user đồng nghĩa với việc thằng user_id trong micropost buộc sẽ phải có